### PR TITLE
Adding splitting linked list program

### PR DESCRIPTION
**Pull request by Mohim-Singla**
issue #687 

Adding program splitting liked list into two parts.

Please review it.

### CODE REVIEW

File: Code/C++/Splitting_Linked_lists.cpp [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+using namespace std;
+
+//initializing structure
+struct node{
+int data;
+node *next;
+};
+
+struct node* head=NULL;
```
Comment (rudrakshi99) [inline Code/C++/Splitting_Linked_lists.cpp:23]: use `class` instead of `struct`

**Review by rudrakshi99 — COMMENTED** [review]
add necessary inline comments.

**Review by nakul-19 — CHANGES_REQUESTED** [review]
@Mohim-Singla please make changes in respective readme.

**Comment by Mohim-Singla** [comment]
@nakul-19 Updated readme.md file and resolved conflicts.

**Comment by rudrakshi99** [comment]
> add necessary inline comments and replace `NULL` with `nullptr`.

Do these changes also.

**Comment by Mohim-Singla** [comment]
@rudrakshi99 done

**Review by rudrakshi99 — APPROVED** [review]

**Pull request merged**